docs/Symbols.md

```diff
@@ -1,5 +1,9 @@
 # Symbols
 
+## xml.symbols.enabled
+
+Enable/disable document symbols (Outline). Default is `true`. No symbols are given if `\"xml.symbol.enabled\": false`.
```

Comment:
At the end, the setting should be `"xml.symbols.enabled"` (plural)

Reply:
fixed